docs/guide/extractors.md

```diff
@@ -0,0 +1,401 @@
+---
+id: extractors
+title: 🔬 Extractors
+description: Understanding how middleware extracts values from HTTP requests
```

Comment:
I believe this line was from when extractors was intended to be its own thing. It should be refactor